### DISCUSSION TITLE
Reseting EEPROM

### DISCUSSION THREAD

**Discussion by FwedFwed** [Jyers/Marlin#381] [Q&A] [answered]
Ok, so my question is probably very basic, but I haven't been able to find the answer to this question: how do you reset the EEPROM?

I've been checking every question in the Jyers's page (yes, I'm a total noob at github, I just created my account, it's my first time asking something, I don't understand most things related to github and how it works), and I came across some very interesting ones: #253 and #69. The first helped me understand the proper procedure to flash the FW as I was very confused between having to flash the board and/or the lcd. Now that's cleared, I understand that the FW here is for the board. Post #69 mentions having to reset the EEPROM after every FW flash, and that's where I'm stuck. How do I do that.

I'm new to all this, I'm doing my best to educate myself on all this stuff and not ask every question that comes to mind. I was able to learn lots of things on many ressources. Sorry if it's not something I'm supposed to ask here also.

**Comment by darco415** [comment]
https://marlinfw.org/docs/gcode/M502.html

**Comment by Cal-LF** [comment, marked as answer]
In Jyres FW the reset eprom is in the control menu near the bottom. "reset default settings"
All firmware has a menu selection to reset eprom. It may be called something different in each FW.

**Reply by joeclifford** [reply]
Make sure you then select the `save configuration` option after the `reset default settings`.

It doesn't save it automatically.

My work flow is normally:

1. Flash firmware
2. `reset default settings`
3. `save configuration`
4. Make my normal changes, e.g, disable power recovery, etc..
5. `save configuration`

**Comment by FwedFwed** [comment]
Great, thanks a lot for the answers.
